docs/topics/feed-exports.rst

```diff
@@ -135,6 +135,35 @@ Here are some examples to illustrate:
 
     -   ``s3://mybucket/scraping/feeds/%(name)s/%(time)s.json``
 
+You can also define other variables.
+To do so, you need to pass the argument to the process with `-a`.
+For example, 
+
+::
+
+   scrapy crawl quotes -a filename=crawled_quotes
+
+Then, you can refer to it as ``%(filename)s`` in your program.
+In addition to that, you can pass explicitly as an argument to ``CrawlerProcess.crawl`` method, as shown below::
+
+::
+
+    from scrapy.crawler import CrawlerProcess
+
+    class MySpider(scrapy.Spider):
+        # Your spider definition
+        ...
+
+    process = CrawlerProcess(settings={
+        "FEEDS": {
+            "%(filename)s.json": {"format": "json"},
+        },
+    })
+
+    process.crawl(MySpider, filename="crawled_quotes")
+    process.start()
+    ...
+
```

Comment:
The prior documentation already states that spider attributes are replaces. What this new documentation does is expand on the ways that you can define such spider attributes. So I think this new documentation does not belong here, but in https://docs.scrapy.org/en/latest/topics/spiders.html#spider-arguments. I think we can consider the first half covered, but maybe mentioning `CrawlerProcess` makes sense, although I would not go as far as providing a complete example for that, I would indicate the `CrawlerProcess` object creation and the `crawl` call at most.

It would be great, however, to include a reference to https://docs.scrapy.org/en/latest/topics/spiders.html#spider-arguments from the existing documentation here. Maybe we could add a ``.. note::`` after the “Any other named parameter gets replaced by …” paragraph like:

```rst
.. note:: :ref:`Spider arguments <spiderargs>` become spider attributes, hence they can also be used as storage URI parameters.
```

Reply:
@Gallaecio 
Thanks for your comment! I've tried to reflect on your suggestion.
I would appreciate it if you could check out and merge this PR when you have a moment🚀